Gemfile

```diff
@@ -84,3 +84,11 @@ end
 gem 'sucker_punch'
 
 gem 'acts_as_votable', '~> 0.10.0'
+
+gem "github_api"
```

Comment:
Playing devil's advocate here, but since this is for a separate feature, should it make sense to include this on this PR?